.github/workflows/ci.yml

```
@@ -47,3 +70,30 @@
               with:
                   name: benchmark-report
                   path: benchmark.txt
+
+            - name: Post PR summary
+              if: github.event_name == 'pull_request'
+              uses: actions/github-script@v6
```

actionlint (1.7.7)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)